qcodes/plots/qcmatplotlib.py

```diff
@@ -14,6 +14,8 @@
 
 from .base import BasePlot
 import qcodes.config
+from qcodes import DataArray
```

Comment:
Please use the fully qualified import `from qcodes.data.data_array import DataArray` otherwise you end up creating circular references that prevent Matplot from being imported into the top level qcodes namespace

Reply:
@jenshnielsen fixed